### PR TITLE
feat(components): #25 - Adds board list component

### PR DESCRIPTION
**Pull request by naveenvignesh5**
## Steps To Implement Feature

1.  Added new component `app-board-list`
2. Connected the store to select and display values in `app-board-list`

### CODE REVIEW

File: web/src/app/components/app-board-form/app-board-form.component.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<ng-container *ngIf="showForm; else addCta;">
```
Comment (ramsunvtech) [inline web/src/app/components/app-board-form/app-board-form.component.html]: we should reduce as much `ngIf`

File: web/src/app/components/app-board-form/app-board-form.component.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { BoardItem } from './../../../common/models/board';
+import { Store } from '@ngrx/store';
+import { Component, Input } from '@angular/core';
+import { AppState } from 'src/store/reducers';
+
+// actions
+import { addBoard } from './../../../store/actions';
+
+export interface AppBoardComponentInterface {
```
Comment (ramsunvtech) [inline web/src/app/components/app-board-form/app-board-form.component.ts]: it has move to separate `interface`

File: web/src/app/components/app-board-form/app-board-form.component.html (hunk 1 above)
Comment (naveenvignesh5) [reply web/src/app/components/app-board-form/app-board-form.component.html]: the component containing it removed from this pr

File: web/src/app/components/app-board-form/app-board-form.component.ts (hunk 2 above)
Comment (naveenvignesh5) [reply web/src/app/components/app-board-form/app-board-form.component.ts]: latest component pushed as the requested format.

**Review by ramsunvtech — APPROVED** [review]

**Pull request merged**